src/lib/constants.ts

```diff
@@ -187,6 +188,13 @@ export const DEFAULT_DEFAULT_ANGLE_UNIT: UnitAngle = 'degrees'
  */
 export const DEFAULT_DEFAULT_LENGTH_UNIT: UnitLength = 'mm'
 
+/**
+ * When no annotation is in the KCL file to specify the defaults
+ */
+export const DEFAULT_DEFAULT_EXPERIMENTAL_FEATURES: WarningLevel = {
```

Comment:
The other is called "default {default length unit}", but this is just "default {experimental features}".

```suggestion
export const DEFAULT_EXPERIMENTAL_FEATURES: WarningLevel = {
```